### ISSUE TITLE
[Bug]: Project PM UI Install button should disable when `null` version selected

### ISSUE THREAD

**Issue by donnie-msft** [NuGet/Home#11096]
### NuGet Product Used

Visual Studio Package Management UI

### Product Version

VS any

### Impact

It bothers me. A fix would be nice

### Repro Steps & Context

Due to a bug in WPF https://github.com/dotnet/wpf/issues/4929,
..NuGet sometimes allows selection of a null separator https://github.com/nuget/home/issues/11007 (see for repro steps).

In Solution PMUI, the Install button is disabled as expected.
Project PM UI is not disabled, but should be. Note that the action does not occur, but no info is given to the user.

![image](https://user-images.githubusercontent.com/49205731/127242611-d2d1bbe5-2607-4d7b-97bc-c56ecfd31536.png)

**Comment by martinrrm** [comment]
Closing: This was fixed with the Floating Versions feature
